runtime/src/accounts_index.rs

```diff
@@ -7,11 +8,11 @@ use std::{
 pub type Slot = u64;
 type SlotList<T> = Vec<(Slot, T)>;
 pub type RefCount = u64;
```

Comment:
nits; I think this no longer is needed?

Reply:
I use it for the return type of ref_count_from_storage

Reply:
Oops, you're right.